js/interactive-guides/step-content.js

```diff
@@ -166,27 +166,30 @@ var stepContent = (function() {
             //console.log("displayType: ", content.displayType);
             switch (content.displayType) {
               case 'fileEditor':
-                var newEditor = editor.create(subContainer, step.name, content);
-                //console.log(newEditor);
-                contentManager.setEditor(step.name, newEditor);
+                editor.create(subContainer, step.name, content).done(function(newEditor){
+                  //console.log(newEditor);
+                  contentManager.setEditor(step.name, newEditor, displayTypeNum);
+                });                
                 break;
               case 'commandPrompt':
                 //console.log("commandPrompt detected");
                 var newCmdPrompt = cmdPrompt.create(subContainer, step.name, content);
-                contentManager.setCommandPrompt(step.name, newCmdPrompt);
+                contentManager.setCommandPrompt(step.name, newCmdPrompt, displayTypeNum);
                 break;
               case 'webBrowser':
-                var newWebBrowser = webBrowser.create(subContainer, step.name, content);
-                contentManager.setWebBrowser(step.name, newWebBrowser);
+                webBrowser.create(subContainer, step.name, content).done(function(newWebBrowser){
+                  contentManager.setWebBrowser(step.name, newWebBrowser, displayTypeNum);
+                });                
                 break;
               case 'fileBrowser':
                 //console.log("fileBrowser type found.");
                 var newFileBrowser = fileBrowser.create(subContainer, content, step.name);
-                contentManager.setFileBrowser(step.name, newFileBrowser);
+                contentManager.setFileBrowser(step.name, newFileBrowser, displayTypeNum);
```

Comment:
Same comment as cmdPrompt here for fileBrowser.

Reply:
Fixed